lib/defines.h

```diff
@@ -250,7 +250,6 @@ static inline void memzero(void *ptr, size_t size)
 #ifdef sun			/* hacks for compiling on SunOS */
 # ifndef SOLARIS
 extern char *strdup ();
-extern char *strerror ();
```

Comment:
`configure.ac` also contains a reference to `strerror()` which I believe can be removed.

Reply:
Thanks!